.gitignore

```diff
@@ -12,3 +12,5 @@ pom.xml.asc
 .lein-failures
 .nrepl-port
 .cpcache/
+.idea/
```

Comment:
you shouldn't have to check in your machine-specific breadcrumbs. this should go in your personal global .gitignore instead of the project's

Reply:
Oopsie daisy, that was indeed the wrong file, sorry!